videos.html

```diff
@@ -0,0 +1,209 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Videos</title>
+    <link rel="stylesheet" href="styles.css" />
+    <script
+      src="https://kit.fontawesome.com/0ec53d8e70.js"
+      crossorigin="anonymous"
+    ></script>
+  </head>
+  <body>
+    <nav class="container">
+      <div class="nav-left container">
+        <!--icons left-->
+        <i class="fa-solid fa-bars-staggered"></i>
+        <!--styled together-->
+        <img
+          src="./images/play.png"
+          alt="play"
+          class="play"
+          width="80"
+          height="80"
+        />
+        <span class="logo-name">MyTube</span>
+      </div>
+      <!--middle icons and search bar-->
+      <div class="nav-middle container">
+        <div class="search-box container">
+          <input type="text" placeholder="Search" />
+          <i class="fa-solid fa-magnifying-glass"></i>
+        </div>
+        <i class="fa-solid fa-microphone"></i>
+      </div>
+      <!--icons right-->
```

Comment:
Good job adding the video.html file however, i think it would be better to render reusable parts of the code dynamically to avoid repetitive code. Repetitive code can be difficult to maintain, as any changes or updates need to be made in multiple places. This can be time-consuming and error-prone, and can increase the risk of introducing bugs or inconsistencies into your codebase. Moreover, Repetitive code can also have a negative impact on the performance of your website. The more code that needs to be loaded and executed, the slower the website may be. This can result in a poor user experience and may also affect your website's search engine rankings.